dependencies/vaticle/repositories.bzl

```diff
@@ -40,8 +41,9 @@ def vaticle_typedb_protocol():
     )
 
 def vaticle_typedb_behaviour():
+    # TODO: Return typedb
     git_repository(
         name = "vaticle_typedb_behaviour",
-        remote = "https://github.com/typedb/typedb-behaviour",
-        commit = "bcc49a3ffd8305f844922a2b36e6f705d9a437de",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_behaviour
+        remote = "https://github.com/farost/typedb-behaviour",
+        commit = "66baa5dd51240380308a97c3ca4baed9de6d59a6",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_behaviour
```

Comment:
I believe both of these are approved now.

Reply:
They are, but I'm waiting for @flyingsilverfin 's review